Client/Frontend/Browser/BrowserViewController/BrowserViewController+Menu.swift

```diff
@@ -11,10 +11,24 @@ import Shared
 extension BrowserViewController {
     func featuresMenuSection(_ menuController: MenuViewController) -> some View {
         VStack(spacing: 0) {
-            VPNMenuButton(vpnProductInfo: self.vpnProductInfo) { vc in
-                (self.presentedViewController as? MenuViewController)?
-                    .pushInnerMenu(vc)
-            }
+//<<<<<<< HEAD
```

Comment:
rebase comment, verify it works correct

Reply:
comments is removed, everything is tested and rebased